packages/manifold/contracts/burnredeemUpdatableFee/IBurnRedeemCoreV2.sol

```diff
@@ -0,0 +1,223 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+/// @author: manifold.xyz
+
+import "@openzeppelin/contracts/token/ERC721/IERC721Receiver.sol";
+import "@openzeppelin/contracts/token/ERC1155/IERC1155Receiver.sol";
+import "@openzeppelin/contracts/utils/introspection/IERC165.sol";
+
+/**
+ * Burn Redeem Core interface
+ */
+interface IBurnRedeemCoreV2 is IERC165, IERC721Receiver, IERC1155Receiver  {
+    error NotAdmin(address);
+    error UnsupportedContractVersion();
+    error InvalidToken(uint256);
+    error InvalidInput();
+    error InvalidTokenSpec();
+    error InvalidBurnSpec();
+    error InvalidData();
+    error TransferFailure();
+    
+    error BurnRedeemDoesNotExist(uint256);
+    error BurnRedeemInactive(uint256);
+
+    error InvalidBurnAmount();
+    error InvalidRedeemAmount();
+    error InvalidPaymentAmount();
+
+    error Inactive();
```

Comment:
New error that emit if `initializeBurnRedeem` is called when `active` is set to `false